internal/api/v1beta1/errors.go

```diff
@@ -14,6 +15,7 @@ var (
 	ErrConflictRequest       = errors.New("already exist")
 	ErrRequestBodyValidation = errors.New("invalid format for field(s)")
 	ErrEmptyEmailID          = errors.New("email id is empty")
+	ErrInvalidEmailID        = errors.New("email id is invalid")
```

Comment:
Where is this being used?

Reply:
Looks like this error returned is different https://github.com/goto/shield/pull/38/files#diff-875939978a63c46968edee87a7dd2766b74441e5df991143afbd911c57cfae9eR94